projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -4209,6 +4235,35 @@ public function render() {
 		$field_width         = $this->get_attribute( 'width' );
 		$class               = 'date' === $field_type ? 'jp-contact-form-date' : $this->get_attribute( 'class' );
 
+		if ( ! empty( $this->get_attribute( 'borderradius' ) ) || is_numeric( $this->get_attribute( 'borderradius' ) ) ) {
+			$this->field_styles = 'border-radius: ' . (int) $this->get_attribute( 'borderradius' ) . 'px;';
```

Comment:
Should we append here as well?

```suggestion
			$this->field_styles .= 'border-radius: ' . (int) $this->get_attribute( 'borderradius' ) . 'px;';
```